blocks/additional-info/__link/additional-info__link.css

```diff
@@ -3,6 +3,13 @@
   line-height: 19px;
 }
 
+@media only screen and (max-width: 320px) {
+  .additional-info__link {
+    font-size: 14px;
```

Comment:
You should specify font-weight everywhere. font-weight 500 looks different the the normal one.